### PR TITLE
v1.0.4

### PR DESCRIPTION
**Pull request by y-guyon**
See #1841 for the corresponding pull request for the libavif 1.0.3 release.

### CODE REVIEW

**Review by vrabaud — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.4] - 2024-01-31
+
```
Comment (wantehchang) [inline CHANGELOG.md:10]: Please add bug numbers/URLs to the three bug fixes if there are bug reports.

**Review by wantehchang — COMMENTED** [review]
Thanks for writing the pull request. I suggest a change.

File: CHANGELOG.md (hunk 1 above)
Comment (wantehchang) [reply CHANGELOG.md:10]: I can't find the bug report for the bug fix at line 13.

https://github.com/AOMediaCodec/libavif/pull/1956 says the fix at line 15 is for b/321189607 and b/321189607.

The fix at line 16 is for https://crbug.com/oss-fuzz/65657.

Comment (vrabaud) [reply CHANGELOG.md:10]: b/320234262 is for https://github.com/AOMediaCodec/libavif/pull/1942

Comment (y-guyon) [reply CHANGELOG.md:10]: Done in https://github.com/AOMediaCodec/libavif/pull/1983 and https://github.com/AOMediaCodec/libavif/pull/1984

Comment (y-guyon) [reply CHANGELOG.md:10]: Rebased

**Review by wantehchang — APPROVED** [review]

**Comment by wantehchang** [comment]
Thank you, Yannis!

**Pull request merged**